worker/backend/backend.go

```diff
@@ -88,6 +113,24 @@ func (b *Backend) Destroy(handle string) (err error) { return }
 // Errors:
 // * None.
 func (b *Backend) Containers(properties garden.Properties) (containers []garden.Container, err error) {
+	var ctx = namespaces.WithNamespace(context.Background(), b.namespace)
+
+	filters, err := propertiesToFilterList(properties)
```

Comment:
TODO: what do we actually use filters for in practice? It looks like we generally retrieve containers from our container server using `handle`, but I wonder if we can push more of the container-retrieving logic out of Concourse DB/web memory and into the container server.